test/index.js

```diff
@@ -1230,15 +1230,22 @@ describe('Underdog', () => {
 
         flags.onCleanup = async () => {
 
-            client.destroy();
+            client.destroy(); // Alternatively, client.close()
             await server.stop();
         };
 
         const request = client.request({ ':path': '/' });
 
+        // Consumes received data, required to trigger req stream's end
+        // See: https://github.com/nodejs/help/issues/650
+        request.on('data', () => {});
```

Comment:
```suggestion
        request.on('data', Hoek.ignore);
```

Reply:
oh, rad, yea, will do, thanks for calling that out. 🙏